pkg/storage/unified/apistore/restoptions.go

```diff
@@ -63,6 +63,7 @@ func NewRESTOptionsGetterMemory(originalStorageConfig storagebackend.Config, sec
 	backend, err := resource.NewKVStorageBackend(resource.KVBackendOptions{
 		KvStore:                      kv,
 		WithExperimentalClusterScope: true,
+		WatchPollInterval:            5 * time.Minute,
```

Comment:
I wonder if this should be exposed as a configuration option, so we can tune this if necessary?